README.md

```diff
@@ -1,27 +1,55 @@
-# privacy-evaluator
+# Privacy Evaluator
+<!--[![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)-->
+
+- space for the description of our project (core idea)
+
+- For example:
+The privML Privacy Evaluator is a tool that assesses ML model's levels of privacy by running different attacks on it. .....
+
+## Dependencies
+- like necessary packages
+- version of packages
+- we have to keep this section up-to-date during development
 [![Python 3.7](https://img.shields.io/badge/python-3.7-blue.svg)](https://www.python.org/downloads/release/python-370/)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
-[![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-The privML Privacy Evaluator is a tool that assesses ML model's levels of privacy by running different attacks on it.
-
-## How does it work?
+## Installation
```

Comment:
In general: Please do not just insert dummy-texts to the project, but fill all the sections out accordingly to how much we know for now. 
If the content of a section is not known yet, please indicate "tbd" (to be done) instead of some dummy-texts.